documentation/advanced/tutorial-i18n-localization.asciidoc

```diff
@@ -169,3 +169,139 @@ public class MyLocale extends Div {
     }
 }
 ----
+
+=== Supporting Right-to-Left mode in the application
+
+Vaadin components have now support for RTL languages, so to make your
```

Comment:
Would leave out the "now", as it implies this is a new feature, which will not be the case in 1 year

Reply:
```suggestion
Vaadin components have support for RTL languages. To make your
```
Vaadin components supporting RTL does not imply that you have to make changes to your application. It’s good to note that they do work out-of-the-box, but it’s a separate task to make your custom application code also support RTL.

Reply:
Changed the text a bit. WDYT?